time stamp
bool[] enable_camera
int32[] sonar_id
float32[] distance
